themes/classic/templates/catalog/_partials/product-variants.tpl

```diff
@@ -35,4 +35,5 @@
       {/if}
     </div>
   {/foreach}
+  <input type="hidden" name="idCombination" id="idCombination" value="{$product.id_product_attribute}">
```

Comment:
@julienbourdeau thank you, I understand what part is confusing.

But need to add `id-product-attribute` value to the _partials/product-variants.tpl file because this part updating after ajax call.

What is the best way to add value instead of <input>? I will prepare new PR.

<section id="id-product-attribute" id-product-attribute="{$product.id_product_attribute}" itemscope itemtype="https://schema.org/Product">
or any another way?